### ISSUE TITLE
Crash with IllegalStateException

### ISSUE THREAD

**Issue by agvxov** [plantuml/plantuml#1934]
Given the following input:
```puml
' @BAKE plantuml crasher.puml
@startuml
a::b -- b
b.b -- c
@enduml
```
plantuml yields:
```cy
java.lang.IllegalStateException
        at net.sourceforge.plantuml.svek.Bibliotekon.getNodeUid(Bibliotekon.java:137)
        at net.sourceforge.plantuml.abel.Link.getEntityPort(Link.java:226)
        at net.sourceforge.plantuml.abel.Link.getEntityPort1(Link.java:216)
        at net.sourceforge.plantuml.svek.SvekEdge.<init>(SvekEdge.java:243)
        at net.sourceforge.plantuml.svek.GraphvizImageBuilder.buildImage(GraphvizImageBuilder.java:226)
        at net.sourceforge.plantuml.svek.CucaDiagramFileMakerSvek.createFileInternal(CucaDiagramFileMakerSvek.java:103)
        at net.sourceforge.plantuml.svek.CucaDiagramFileMakerSvek.createFile(CucaDiagramFileMakerSvek.java:69)
        at net.atmp.CucaDiagram.exportDiagramInternal(CucaDiagram.java:455)
        at net.sourceforge.plantuml.classdiagram.ClassDiagram.exportDiagramInternal(ClassDiagram.java:83)
        at net.sourceforge.plantuml.UmlDiagram.exportDiagramNow(UmlDiagram.java:138)
        at net.sourceforge.plantuml.AbstractPSystem.exportDiagram(AbstractPSystem.java:207)
        at net.sourceforge.plantuml.PSystemUtils.exportDiagramsDefault(PSystemUtils.java:207)
        at net.sourceforge.plantuml.PSystemUtils.exportDiagrams(PSystemUtils.java:96)
        at net.sourceforge.plantuml.SourceFileReaderAbstract.getGeneratedImages(SourceFileReaderAbstract.java:190)
        at net.sourceforge.plantuml.Run.manageFileInternal(Run.java:510)
        at net.sourceforge.plantuml.Run.processArgs(Run.java:404)
        at net.sourceforge.plantuml.Run.manageAllFiles(Run.java:371)
        at net.sourceforge.plantuml.Run.main(Run.java:206)
```
The output should be anything, but a crash.

OS: 6.7.2-gentoo-r1 GNU/Linux
Version:
```yaml
PlantUML version 1.2024.7 (Sat Sep 07 13:18:17 CEST 2024)
(GPL source distribution)
Java Runtime: OpenJDK Runtime Environment
JVM: OpenJDK 64-Bit Server VM
Default Encoding: UTF-8
Language: en
Country: US

PLANTUML_LIMIT_SIZE: 4096

Dot version: dot - graphviz version 10.0.1 (20240210.2158)
Installation seems OK. File generation OK
``` 

I would gladly try on the newest version, but im not sure how. trying to plainly run java on the "build/libs/plantuml-1.2024.8beta1.jar" gradlew created, java complains that it can find no entry point.

**Comment by selincos** [comment]
I tried on the newest version and it did not throw an exception. Could you please try and close if it is fixed?

**Comment by agvxov** [comment]
surely, but as i mentioned at the end of the OP, im not exactly sure how. im not a java person.
could i get a bit of guidance on what to do after compilation?

**Comment by The-Lum** [comment]
Hi all,

_[This is an Issue Review]_ 👀

This is now fixed, on [v1.2024.8](https://github.com/plantuml/plantuml/releases/tag/v1.2024.8)
- Thanks @agvxov for the report.
- Thanks PlantUML team and @arnaudroques for your work.

Regards.
